src/main/java/net/sf/jabref/util/Util.java

```diff
@@ -665,6 +664,66 @@ public static String getResultsWithEncoding(URLConnection source, Charset encodi
     }
 
     /**
+     * Get the results of HTTP post on a URL and return contents as a String.
+     *
+     * @param source postData encoding
+     * @return
+     * @throws IOException
+     */
+    public static String getPostResultsWithEncoding(URL source, String postData, Charset encoding) throws IOException {
```

Comment:
I left it in Util for now, since this is where I saw all the other URL accesses.  But I suppose future work would be to move all these to URLUtil?